bioimageio/spec/model/v0_4/schema.py

```diff
@@ -166,9 +166,23 @@ def matching_halo_length(self, data, **kwargs):
             raise NotImplementedError(type(shape))
 
 
-class PytorchStateDictWeightsEntry(_WeightsEntryBase):
+class _WeightsEntryBase(_WeightsEntryBase03):
     raw_nodes = raw_nodes
+    dependencies = fields.Dependencies(
+        bioimageio_description="Dependency manager and dependency file, specified as `<dependency manager>:<relative "
```

Comment:
Can we extend the description here, to make clear that this is only for this weight format?

Reply:
I added: "These dependencies are only valid in the context of the specified weight format."